pacman/operations/rigged_algorithms/random_placer.py

```diff
@@ -0,0 +1,97 @@
+from pacman.utilities.utility_objs import ResourceTracker
+from pacman.utilities.algorithm_utilities import placer_algorithm_utilities
+from pacman.model.placements import Placement, Placements
+from pacman.model.constraints.placer_constraints import SameChipAsConstraint
+
+from spinn_utilities.progress_bar import ProgressBar
+
+import random as default_random
```

Comment:
why rename this? why not rename your variable so that its more readable and keep random as a known import